docs/src/usage.md

```diff
@@ -0,0 +1,12 @@
+# Usage
+
+At present, Kani can used in two ways:
+ * [On a single file](./kani-single-file.md) with the `kani` command.
+ * [On a package](./cargo-kani.md) with the `cargo-kani` command.
+
+Running [Kani on a single file](./kani-single-file.md) is quite useful for small examples.
```

Comment:
or projects that don't use cargo... like Rust for Linux. :)

Reply:
That said, we still need to fix linking so users can actually use this with makefile or other build systems.

Reply:
That said, we still need to fix linking so users can actually use this with makefile or other build systems.